File: Codility/CyclicRotation.py
def solution(A, K):
    answer = ''
    if len(A) == 0 or len(A) == K:
        answer = A
    else:
        arr = A
        for x in range(K):
            valueMoved = arr.pop()
            arr.insert(0, valueMoved)
        answer = arr
    return answer

File: Codility/CyclicRotation.js
function solution(A, K) {
    let answer;
    if (A.length == 0 || A.length == K) {
        answer = A;
    } else {
        let arr = A;
        for (let i = 0; i < K; i++) {
            let valueMoved = arr.pop();
            arr.unshift(valueMoved);
        }
        answer = arr;
    }
    return answer;
}